Framework/CurveFitting/src/Functions/CubicSpline.cpp

```diff
@@ -41,9 +36,9 @@ CubicSpline::CubicSpline()
   declareAttribute("x1", Attribute(1.0));
   declareAttribute("x2", Attribute(2.0));
 
-  declareParameter("y0", 0);
-  declareParameter("y1", 0);
-  declareParameter("y2", 0);
+  declareParameter("Y0", 0);
```

Comment:
I think this change could break existing code if a script contains a function definition that sets these parameters. Is it necessary?